controllers/cesregistry/dogu_registrator_test.go

```diff
@@ -291,7 +342,39 @@ func TestEtcdDoguRegistrator_RegisterDoguVersion(t *testing.T) {
 		require.Error(t, err)
 		assert.ErrorIs(t, err, assert.AnError)
 		assert.ErrorContains(t, err, "failed to check if dogu is already installed and enabled")
-		mock.AssertExpectationsForObjects(t, registryMock, doguRegistryMock)
+	})
+
+	t.Run("fail because the dogu is not enabled", func(t *testing.T) {
```

Comment:
Does this fail during an upgrade? The version would be a hint of that... Maybe add it as context to the test description.